src/ts/components/transaction/Send.tsx

```diff
@@ -124,9 +124,37 @@ class Send extends React.Component<ISendProps, ISendState> {
 
     signExtrinsic(extrinsic, currentAddress, signOptions).then(signature => {
       extrinsic.addSignature(currentAddress as any, signature, signOptions.nonce)
-      this.api.rpc.author.submitAndWatchExtrinsic(extrinsic, (result: SubmittableResult) => {
+
+      const txItem: ITransaction = {
+        txHash: extrinsic.hash.toHex(),
+        from: currentAddress,
+        to: this.state.toAddress,
+        amount: this.state.amount,
+        unit: this.state.siUnit,
+        type: 'Sent',
+        createTime: new Date().getTime(),
+        status: 'Pending',
+        fee: this.state.fee
```

Comment:
not sure if we want to save fee unit or it will be always default to `m`

Reply:
Agree. The fee part needs some improvement.